src/Components/ModelledPropertyBuilder/ModelledPropertyBuilder.model.ts

```diff
@@ -372,7 +371,7 @@ export const mergeTagsAndMapTwinIdsToModelIds = async (
                 twinModelIdMap.set(twinId, modelId);
             });
 
-        for (const [aliasTag, aliasTwinId] of Object.entries(aliasedTwinMap)) {
+        for (const [aliasTag, aliasTwinId] of Object.entries(aliasedTwinIds)) {
```

Comment:
Don't we use the alias tags here?  If we're changing aliasTagMap to a list of just Ids, It seems like this will break.  I'm pretty sure this method needs to access the actual alias to construct paths like `temperatureTag.temperature` where `temperatureTag` is the alias.